Scripts/Items/Functional/Automaton/KotlAutomaton.cs

```diff
@@ -173,13 +173,13 @@ public KotlAutomaton(Serial serial)
         public override double WeaponAbilityChance { get { return 0.33; } }
 
         public override bool IsScaredOfScaryThings { get {  return false;  } }
-        public override bool IsScaryToPets { get { return true; } }
+        public override bool IsScaryToPets { get { return true || !Controlled; } }
```

Comment:
remove the 'true' so its says:
public override bool IsScaryToPets { get { return !Controlled; } }